src/plugins/esql_datagrid/public/data_grid.tsx

```diff
@@ -47,7 +59,8 @@ const DataGrid: React.FC<ESQLDataGridProps> = (props) => {
   const [activeColumns, setActiveColumns] = useState<string[]>(
     (props.initialColumns || (props.isTableView ? props.columns : [])).map((c) => c.name)
   );
-  const [rowHeight, setRowHeight] = useState<number>(5);
+  const [rowHeight, setRowHeight] = useState<number>(props.initialRowHeight ?? 5);
+  const [rowsPerPage, setRowsPerPage] = useState(10);
```

Comment:
this constant is used here and then again in line 212. Could we export it to the variable? The one above too:
```
 const DEFAULT_INITIAL_ROW_HEIGHT = 5
 const DEFAULT_ROWS_PER_PAGE = 10

Reply:
Sure https://github.com/elastic/kibana/pull/186875/commits/e5f0fdc35ca5d6be0ef96e67189cabeb16d09a8c